com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -663,10 +666,6 @@ private static void mainInternal(String[] args) throws IOException, CompileExcep
             project.setOption("texture-compression", texCompression);
         }
 
-        if (cmd.hasOption("use-vanilla-lua")) {
```

Comment:
No point in explicitly setting these values here since we actually copy over all options to the project here:

https://github.com/defold/defold/pull/6310/files#diff-ea0d2982212f1d66d942bc5b6fe2aa39bf5f8e04a598c033fde540ecad14d6e3L595